framework/src/modules/chain/modules/blocks/process.js

```diff
@@ -588,7 +571,7 @@ Process.prototype.generateBlock = function(keypair, timestamp, cb) {
 					keypair,
 					timestamp,
 					previousBlock: modules.blocks.lastBlock.get(),
-					transactions: ready,
+					transactions: readyTransactions,
 				});
 			} catch (e) {
```

Comment:
We dont need to have a try..catch block here..

Reply:
are you sure? original implementation also has in try catch.

Reply:
ah I see now because of the next catch